kuttl-test.yaml

```diff
@@ -4,6 +4,7 @@ crdDir: ./tests/_build/crds/
 artifactsDir: ./tests/_build/artifacts/
 commands:
   - script: kubectl create namespace jaeger-operator-system 2>&1 | grep -v "already exists" || true
+  - command: kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/release-0.9/manifests/setup/prometheus-operator-0servicemonitorCustomResourceDefinition.yaml
```

Comment:
Instead of doing this, would it make sense to create a Make target called `deploy-prometheus-operator`? Something similar to what we do with `kafka`

Reply:
Well, I don't required the operator here, only the CRD, It will make sense when we have tests about the prometheus use case.

Reply:
Yes but doing this we:
- Allow the developers to install the Prometheus Operator in their environments if they need it (just for development purposes, not running the E2E tests)
- Allow skipping the installation of the Prometheus Operator if we want to use the provided by OLM